docs/Performance/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Performance
+permalink: /performance/
+has_children: true
+nav_order: 6
+---
+
+## Performance
+
+This chapter focuses on asynchronous backend OData calls. It shows how you can make your OData calls "Promise ready".
+
+### Normal ECMAScript 5 Projects
+
+> IE11 compatible
```

Comment:
there's so much good stuff already in this chapter that I'm afraid the `IE 11` mention might indicate to people that the content in this chapter is "old" or "IE 11" only. Rewording this not to be IE11-centric should do justice to the excellent content.

Reply:
I just dropped the note about IE 11 :)